### ISSUE TITLE
Items falling out of crafting terminal

### ISSUE THREAD

**Issue by MineCon1987** [tom5454/Toms-Storage#256]
I was in the nether, crafted some golden apples and when I removed the recipe from the crafter and just so happened to visit the over world I saw the items I was just using on the floor by my terminal.

I believe this is also how I lost a shulker full of unbreaking 3 books.

My storage system is no where near full

**Comment by tom5454** [comment]
If your inventory is full and the system can't place items in, it will drop, check the chunkloading around your base (Could be #255), make sure all of your chests are loaded.

**Comment by MineCon1987** [comment]
Yeah I furst thought it was a full inventory. But my stroge is in my spawn
chunks and my system was only half full.
